pipeline/src/node_properties/track_properties.rs

```diff
@@ -20,3 +22,19 @@ pub enum Tracker {
     /// bounding boxes across frames to determine the object’s unique ID.
     Iou,
 }
+
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "snake_case")]
+pub enum TrackerProfile {
+    /// Loads the default tracker configuration
+    Default,
+
+    /// If available for this tracker type, loads a configuration tuned for stationary objects
+    Stationary,
```

Comment:
Is it available for all tracker types? Probably it is better to put tracker profiles under certain tracker variants.

```rust
pub enum Tracker {
    Dcf(DcfTracker),
    Klt,
    Iou,
}

pub struct DcfTracker {
    pub profile: DcfTrackerProfile,
}

pub enum DcfTrackerProfile {
   ...
}
```

Reply:
took this more generic approach mainly because I was advised there's a limitation in the console side, it can't handle 2nd level options to just a specific tracker type, turning that all of them will have the Profile option